astropy/coordinates/representation.py

```diff
@@ -223,6 +223,24 @@ def _unitstr(self):
                            for component in self.components]))
         return unitstr
 
+    def __eq__(self, other):
```

Comment:
Just to be sure: do we really need this? I ask in part because I think a representation is more like an array, and if I compared two instances, I would thus expect to get an array of bool back.